_data/simple-icons.json

```
@@ -13220,6 +13220,11 @@
             "hex": "026938",
             "source": "https://www.saudiatechnic.com/media-center/press-kit"
         },
+        {
+            "title": "Saxo Bank",
+            "hex": "111827",
```

As mentioned in https://www.home.saxo/about-us/media-center, we should use `#051478`
```suggestion
            "hex": "051478",
```